custom_account_setup.cdc

```diff
@@ -0,0 +1,31 @@
+import NonFungibleToken from 0xf8d6e0586b0a20c7
```

Comment:
I am not sure putting this at the root of the project is the best place for it - maybe we should have a separate directory for cadence code? @krnjn what do you think?

Reply:
That sounds good to me.